### ISSUE TITLE
exclude echo-style returns

### ISSUE THREAD

**Issue by tizu69** [tomarrell/wrapcheck#58]
Echo (that web framework) uses error as its return value for route handlers.

```go
return c.String(http.StatusForbidden, "You have no more invites.")
```

Something like this probably shouldnt cause the linter to throw;

```
error returned from interface method should be wrapped: sig: func (github.com/labstack/echo/v4.Context).String(code int, s string) error (wrapcheck)
```

**Comment by tizu69** [comment]
On the other hand I've noticed that something like

```go
if _, err := res.UserFromGithub(d.I, data.Name, user.ID); err == nil {
	return echo.NewHTTPError(http.StatusInternalServerError,
		"Something went wrong.").SetInternal(err)
}
```

does not properly throw the wrapping warning.
